com.unity.render-pipelines.high-definition/Documentation~/Mask-Map-and-Detail-Map.md

```diff
@@ -7,7 +7,12 @@ This document describes the format of the mask map and detail map so that you ca
 To create a mask map:
 
 1. Open image editing software that supports channel editing (such as Adobe Photoshop).
-2. Drag your grayscale textures into their respective color channel. For information about which texture belongs in which channel, see [mask map](#MaskMap) and [detail map](#DetailMap).<br />![](Images/MaskMapAndDetailMap1.png)
+2. Import your grayscale textures onto separate layers.
+3. Make only one of the layers visble.
+4. Under **Adjustments**, select **Curves**. 
+5. Leaving only the color channel you want that texture to be in, set the outputs of the other channels to 0 and the inputs to 4. For information about which texture belongs in which channel, see [mask map](#MaskMap) and [detail map](#DetailMap).<br />![](Images/MaskMapAndDetailMap1.png)
+6. Merge the curve with layer you've just worked on.
```

Comment:
Did we create a layer on which to apply curves? Usually in photoshop you just apply the curves in the window. I'm not sure why we're talking layers here.